src/packages/accessibility-insights-ui/root/README.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+# accessibility-insights-ui
+
+This library is published was created so the Accessibility Insights website can share the overall
```

Comment:
I'm not sure if I'm reading this wrong but this part:
```
... is published was...
```
looks odd to me. Is this right?

Reply:
I have no idea what I was thinking here. Let me revise. :)

Reply:
Fixed the typos.